src/apps/components/AppAlerts/AppRowDisabledAlert.tsx

```diff
@@ -0,0 +1,59 @@
+import { AppPaths } from "@dashboard/apps/urls";
+import Link from "@dashboard/components/Link";
+import { StopPropagation } from "@dashboard/components/StopPropagation";
+import { AppListItemFragment } from "@dashboard/graphql";
+import { DisabledIcon } from "@dashboard/icons/Disabled";
+import { Box, Text, Tooltip } from "@saleor/macaw-ui-next";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+interface AppRowDisabledAlertProps {
+  app: AppListItemFragment;
+}
+
+export const AppRowDisabledAlert = ({ app }: AppRowDisabledAlertProps) => {
+  if (app.isActive) {
+    return null;
+  }
+
+  const detailsLink = AppPaths.resolveAppDetailsPath(app.id);
+
+  return (
+    <Tooltip>
+      <Tooltip.Trigger>
+        <Box color="default2" __transform="scale(0.8)">
+          <DisabledIcon />
+        </Box>
+      </Tooltip.Trigger>
+
+      <Tooltip.Content align="start" side="bottom">
+        <StopPropagation>
```

Comment:
question: why do you use `StopPropagation` ?

Reply:
Because the event is transffered to the app list row and when you click on the link inside the tooltip you would get navigated to the app instead of the link.